core_perception/ekf_localizer/include/ekf_localizer/ekf_localizer.h

```diff
@@ -57,7 +57,6 @@ class EKFLocalizer
   ros::Subscriber sub_pose_with_cov_;    //!< @brief measurement pose with covariance subscriber
```

Comment:
Please revert all the changes to this package except the changes to the call to broadcastTF, timer_tf_, tf_rate_

Reply:
sounds good.